gcn/notices/chime/subsequent.example.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "subsequent",
```

Comment:
This is also not a valid value for `alert_type`, would `update` make sense here?